src/components/app/app.scss

```diff
@@ -1,3 +1,10 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500");
```

Comment:
please move this import to the to of the html

Reply:
BTW buttons should be Cancel and Ok (please also rename methods in js according to this changes)

Reply:
done